tests/fake_api.py

```diff
@@ -509,7 +509,7 @@ def openshift(request):
 @pytest.fixture(params=[{'kwargs': None, 'additional_config': None, 'platform_descriptors': None}])
 def osbs(request, openshift):
     kwargs = request.param['kwargs'] or {}
-    platform_descriptors = request.param['platform_descriptors'] or {}
+    platform_descriptors = request.param.get('platform_descriptors') or {}
```

Comment:
request.param.get('platform_descriptors', {})

Reply:
That's not working in my tests, I don't know why.

Reply:
```
        if platform_descriptors is None:
            print("params: {0} pds {1} adds {2}".format(request.param, request.param.get('platform_descriptors', {}), request.param.get('additional_config', {})))
            raise
```
returns
`params: {u'additional_config': None, u'platform_descriptors': None, u'kwargs': None} pds None adds None`

I don't get it.

Reply:
nvm leave what you have, I missed that the key is actually defined but set to None,
so in that case it won't use that value, since key exists